packages/secure_storage/amplify_secure_storage_dart/Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: ffi_bindings_linux
+.PHONY: ffi_bindings_macos
+
+ffi_bindings_linux:
+	dart run ffigen --config ffigen.glib.config.yaml
```

Comment:
I would add `git submodule update --init` at the start of these

Reply:
good call. updated.